convert/Cargo.toml

```diff
@@ -18,11 +18,16 @@ protobuf = { version = "3.2.0", optional = true }
 rmp-serde = { version = "1.1.2", optional = true }
 serde = "1.0.186"
 serde_json = "1.0.105"
+extism-convert-macros.workspace = true
 
 [dev-dependencies]
+# Only required for tests run from the workspace root, as that enables the `extism-path` feature.
+extism.workspace = true
```

Comment:
Alternatively, we could add a hidden `# mod extism {pub use extism_convert as convert;}` to the doc tests.